.github/workflows/check-changes-crd-api.yaml

```
@@ -42,7 +42,7 @@ jobs:
       - name: Apply Changes
         uses: EndBug/add-and-commit@v9
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.CLOUOPERATOR_REPO_WRITE }}
         with:
           author_name: CRD API Docs Bot
           author_email: crd_api_docs_bot@github.com
```

Do we need to set author details, when using the PAT? If so let's update to match the underlying user:
```suggestion
          author_name: Cloud Operator
          author_email: 169066274+cloud-operator@users.noreply.github.com
```

---

good idea